stable/mission-control/values.yaml

```diff
@@ -118,27 +120,14 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-## Common
-## Create secret with keys and certs as per documentation provided in README.md
-existingCertsSecret:
-
-## Create secret for certs from provided values. Follow script given in README.md to generate keys and certs. Example is given in
-## Example values given in https://github.com/kubernetes/charts//blob/master/incubator/mission-control/ci/test-values.yaml
-insightKey:
-insightCrt:
-jfmcCrt:
-jfmcKeystore:
-jfmcTruststore:
-jfmcStorePasswd:
-
 missionControl:
   replicaCount: 1
   name: mission-control
   appName: jfmc-server
   home: /var/opt/jfrog/mission-control
   image: docker.bintray.io/jfrog/mission-control
   ## Note that by default we use appVersion to get image tag
-  # version: 3.2.0
+  # version: 3.3.0
```

Comment:
Please remove the `3.3.0`. Leave is just `# version:`. Easier to support later on. Do it for the other services too.
We should have the app version written in the `Chart.yaml` only.